### ISSUE TITLE
Speech commands v1

### ISSUE THREAD

**Issue by yunzqq** [s3prl/s3prl#485]
The preprocessing for speech commands v1 is at /s3prl/downstream/speech_commands/expert.py, right?

**Comment by wind-apprentice** [comment]
I assume that "preprocessing for speech commands" means preparing the dataset and dataloader for the downstream task.
If this is the case, then that's true. You may take a look at expert.py and then you can find out that there are several functions related to preparing the data for the speech commands task, such as a function called split_dataset which helps us split the dataset into training set and validation set.
@dlion168
